From dcc109ad172820d15ebcb1cf8294c19c354cdabf Mon Sep 17 00:00:00 2001
From: zrj <rimvydas.jasinskas@gmail.com>
Date: Thu, 22 Aug 2019 10:10:15 +0300
Subject: [PATCH 30/52] libc: Remove a single use of vm_offset_t type.

 Use (u_long) for casting (void *) like it is already used elsewhere for
 consistency.
---
 lib/libc/sysvipc/shm.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/libc/sysvipc/shm.c b/lib/libc/sysvipc/shm.c
index a90b008a72..a53de824d4 100644
--- a/lib/libc/sysvipc/shm.c
+++ b/lib/libc/sysvipc/shm.c
@@ -244,8 +244,8 @@ sysvipc_shmat(int shmid, const void *shmaddr, int shmflg)
 	flags = MAP_SHARED;
 	if (shmaddr) {
 		if (shmflg & SHM_RND) {
-			addr = (void *)((vm_offset_t)shmaddr & ~(SHMLBA-1));
-		} else if (((vm_offset_t)shmaddr & (SHMLBA-1)) == 0) {
+			addr = (void *)((u_long)shmaddr & ~(SHMLBA-1));
+		} else if (((u_long)shmaddr & (SHMLBA-1)) == 0) {
 			addr = __DECONST(void *, shmaddr);
 		} else {
 			errno = EINVAL;
-- 
2.22.0

